opentelemetry-instrumentation/setup.cfg

```diff
@@ -43,6 +43,7 @@ include_package_data = True
 install_requires =
     opentelemetry-api == 1.4.0.dev0
     wrapt >= 1.0.0, < 2.0.0
+    yarl ~= 1.6
```

Comment:
I think this dependency can be avoided by using `urllib.parse`:

``` python
In [57]: from urllib.parse import urlparse, urlunparse

In [58]: parsed_url = urlparse("http://username:password@httpbin.com")

In [59]: urlunparse((parsed_url.scheme, ":".join((parsed_url.hostname, str(parsed_url.port or ""))), parsed_url.path, parsed_url.params, parsed_url.query, parsed_url.fragment))
Out[59]: 'http://httpbin.com:'
```

Reply:
Figured that the yarl implementation is more concise and is utilized by aiohttp but it makes sense to limit dependencies. Just updated with `urllib.parse`.